internal/controllers/suite_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package new_controllers
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/node-feature-discovery-operator/internal/test"
+	//+kubebuilder:scaffold:imports
```

Comment:
Just a question to educate myself: what is this doing? 😇 Can you point some documentation?

Reply:
This marker is set by kubebuilder during project bootstrap.  So it actually appears only in file that are created during bootstrap, but is missing from the files that are created later by developers. It's purpose is to manage the import of the Go file ( go over the code and adding the imports as needed), but i have never seen it used, since developers usually do the imports as part of developing code.  I can remove it, if you think it clutters the code

Reply:
Your call, I think we can keep it if/when it's "standard clutter". I only wanted to understand better as this is the first time seeing this marker

Reply:
ok, than i will keep for now, unless we get additional complaints